### PR TITLE
On hidden issues, Request access feature 

### PR DESCRIPTION
**Pull request by HanilJain**
### Fixed #1090 

- on accessing hidden issues it will direct them to page(issue/<int:pk>/request_access/)
  with mentioning < issue has been hidden by the user that reported it.>

- on requesting access it will email to the user for the access

### CODE REVIEW

**Comment by HanilJain** [comment]
## Additional Feature to be added 

which will be fixed in this issue #1685 

#### Feature description 

- individual user can see their hidden issues in dedicated page to it 
- user could edit access to issues and user you can view it

File: blt/urls.py [hunk 1]
```diff
@@ -317,7 +318,7 @@
     path("issue/<str:issue_pk>/comment/update/<str:comment_pk>/", website.views.update_comment, name="update_comment"),
     # delete_comment 
     path("issue2/comment/delete/", website.views.delete_comment, name="delete_comment"),
-    re_path(r"^issue/(?P<slug>\w+)/$", IssueView.as_view(), name="issue_view"),
+    path("issue/<int:slug>/" , IssueView.as_view() , name = "issue_view"),
```
Comment (DonnieBLT) [inline blt/urls.py:321]: Can keep this how it is

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
This looks similar to the other whole request. Can you please close the one that is not required and only keep the one that has all the changes required open?

**Comment by HanilJain** [comment]
closing this pull request as all additional feature including this have been added in #1737

**Pull request closed without merging**